src/components/course/data/hooks.jsx

```diff
@@ -405,13 +405,13 @@ export const useExtractAndRemoveSearchParamsFromURL = () => {
  * a imperceivable delay is introduced to allow enough time for analytic event request to resolve.
  *
  * @param {object} args
- * @param {string} args.href If click handler is used on a hyperlink, this is the destination url.
+ * @param {string} args.href (optional) If click handler is used on a hyperlink, this is the destination url.
  * @param {string} args.eventName Name of the event
  *
  * @returns Click handler function for clicks on buttons, external hyperlinks (with a delay), and
  * internal hyperlinks (e.g., using ``Link``).
  */
-export const useTrackSearchConversionClickHandler = ({ href, eventName }) => {
+export const useTrackSearchConversionClickHandler = ({ href = undefined, eventName }) => {
```

Comment:
[nit/curious] If `href` isn't specified, does it fallback to `undefined` already, even without this default function argument of `href = undefined`?